packages/event-dispatcher/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@solid-primitives/event-dispatcher",
+  "version": "0.0.100",
+  "description": "A template primitive example.",
+  "author": "Your Name <you@youremail.com>",
+  "contributors": [],
+  "license": "MIT",
+  "homepage": "https://github.com/solidjs-community/solid-primitives/tree/main/packages/event-dispatcher#readme",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/solidjs-community/solid-primitives.git"
+  },
+  "bugs": {
+    "url": "https://github.com/solidjs-community/solid-primitives/issues"
+  },
+  "primitive": {
+    "name": "event-dispatcher",
+    "stage": 0,
+    "list": [
+      "createPrimitiveTemplate"
+    ],
+    "category": "Display & Media"
+  },
+  "private": false,
+  "sideEffects": false,
+  "type": "module",
+  "files": [
+    "dist"
+  ],
+  "main": "./dist/server.cjs",
+  "module": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "exports": {
+    "node": {
+      "import": "./dist/server.js",
+      "require": "./dist/server.cjs"
+    },
```

Comment:
The exports mention a `server` export, but one isn't built.
It's important for the packages to work fine in SSR so using any client-only APIs (such as CustomEvent) should happen only in a client-only runtime.
To add one you only need to have a `server.ts` file in src with server fallback primitives. Tsup already has the correct setup.
See this as an example: https://github.com/solidjs-community/solid-primitives/blob/main/packages/devices/src/server.ts
Additionally, you can add a test for matching exports between `server` and `index`: https://github.com/solidjs-community/solid-primitives/blob/main/template/test/exports.test.ts